charts/argo-cd/templates/NOTES.txt

```diff
@@ -9,7 +9,8 @@ In order to access the server UI you have the following options:
       - Add the `--insecure` flag to `server.extraArgs` in the values file and terminate SSL at your ingress: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/ingress.md#option-2-multiple-ingress-objects-and-hosts
 
 
-After reaching the UI the first time you can login with username: admin and the password will be the
-name of the server pod. You can get the pod name by running:
+After reaching the UI the first time you can login with username: admin and the random password generated during the installation. You can find the password by running
```

Comment:
Should there be a version check (global.image.tag maybe?) to match Argo CD's version or is this (a sensible default) enough?

Reply:
I think we can proceed with the new text only. The chart probably only works with 2.x now. At least for Dex there are command arguments which doesn't work for pre 2.x